artifactory/commands/python/setuppy.go

```diff
@@ -17,87 +20,88 @@ func ExtractPackageNameFromSetupPy(setuppyFilePath, pythonExecutablePath string)
 	}
 
 	// Extract project name from file content.
-	return getProjectNameFromFileContent(content)
+	return getProjectIdFromFileContent(content)
 }
 
-// Get package-name from PKG-INFO file content.
-// If pattern of package-name not found, return an error.
-func getProjectNameFromFileContent(content []byte) (string, error) {
+// Get package ID from PKG-INFO file content.
+// If pattern of package name of version not found, return an error.
+func getProjectIdFromFileContent(content []byte) (string, error) {
 	// Create package-name regexp.
 	packageNameRegexp, err := utils.GetRegExp(`(?m)^Name\:\s(\w[\w-\.]+)`)
 	if err != nil {
 		return "", err
 	}
 
-	// Find first match of packageNameRegexp.
-	match := packageNameRegexp.FindStringSubmatch(string(content))
-	if len(match) < 2 {
+	// Find first nameMatch of packageNameRegexp.
+	nameMatch := packageNameRegexp.FindStringSubmatch(string(content))
+	if len(nameMatch) < 2 {
 		return "", errorutils.CheckErrorf("Failed extracting package name from content.")
 	}
 
-	return match[1], nil
+	// Create package-version regexp.
+	packageVersionRegexp, err := utils.GetRegExp(`(?m)^Version\:\s(\w[\w-\.]+)`)
```

Comment:
Compiling the same regexp more than once during the life-time of an application should be avoided, becuase it's en expensive action.